PeripheryFunctions/TS_GetIDs.m

```diff
@@ -67,29 +73,27 @@
 case 'ops'
     theDataTable = Operations;
 otherwise
-    error('Specify ''ts'' or ''ops''');
+    error('Specify ''ts'', ''ops'', or ''opsName''');
```

Comment:
What's 'opsName'? This choice doesn't actually have any functionality.

Reply:
Outdated print -- can be removed since nameOrKeywords is there now.